ckanext/data_qld/reporting/controller.py

```diff
@@ -42,7 +42,7 @@ def index(self):
 
             extra_vars = {
                 'organisations': organisations,
-                'user_dict': get_action('user_show')({}, {'id': toolkit.c.userobj.id})
+                'user_dict': get_action('user_show')({}, {'id': c.userobj.id})
```

Comment:
This is still passing an empty context dict, which will fail when used in conjunction with `ckanext-qgov`

Reply:
@ThrawnCA have you seen the explanation above regarding your other comments on empty context dicts?

Reply:
lets try and deploy this to dev where we have all plugins and see how it goes.

Reply:
we have the `ckan-ex-qgov` extension installed on our local development environments and on the QA server, running on the QOL CKAN fork